lib/tasks/event_log.rake

```diff
@@ -0,0 +1,18 @@
+namespace :event_log do
+  desc "Export the event log as a CSV"
+  task :export, %i[min_id max_id] => :environment do |_, args|
+    USAGE_MESSAGE = "usage: rake event_log:export[<min_id>, <max_id>]".freeze
+    abort USAGE_MESSAGE unless args[:min_id] && args[:max_id]
+
+    path = "#{ENV['GOVUK_APP_ROOT'] || Rails.root}/reports"
+    EventLogPresenter.new(min_id, max_id).write_csv(path)
+
+    puts "Report successfully generated inside #{path}"
+  end
+
+  desc "Delete all events in the event log older than 2 years"
+  task delete_logs_older_than_two_years: :environment do
+    delete_count = EventLog.where("created_at < ?", 2.years.ago).delete_all
```

Comment:
That will delete everything up to two years ago.

Reply:
Yup, that's what we're trying to achieve. From a data retention perspective, we're only keeping audit events for the last two years.

Reply:
What am I missing? The query says less than 2 years old.

Reply:
It's saying anything before 2 years ago...

Reply:
"Where the the created at timestamp is less than (further back in time than) the date two years ago."

Reply:
Time for me to go home. 🤦‍♂